test/SourceKit/DocSupport/doc_internal_closure_label.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t.module-cache)
+//
+// RUN: %target-swift-frontend -typecheck -emit-module-interface-path %t/label.swiftinterface -enable-library-evolution -module-cache-path %t.module-cache %s
+// RUN: %FileCheck --check-prefix=SWIFT_INTERFACE %s < %t/label.swiftinterface
+// RUN: %sourcekitd-test -req=doc-info -module label -- -I %t -module-cache-path %t.module-cache > %t.response
```

Comment:
I believe this fixes the test failure.
```suggestion
// RUN: %sourcekitd-test -req=doc-info -module label -- -I %t -target %target-triple -module-cache-path %t.module-cache > %t.response
```

Reply:
I see. Thanks for the fix suggestion!